testing/util/random/epbs.go

```diff
@@ -119,8 +120,12 @@ func AttestationData(t *testing.T) *ethpb.AttestationData {
 
 // Deposit creates a random Deposit for testing purposes.
 func Deposit(t *testing.T) *ethpb.Deposit {
+	proof := make([][]byte, 33)
+	for i := 0; i < 33; i++ {
+		proof[i] = randomBytes(32, t)
+	}
 	return &ethpb.Deposit{
-		Proof: [][]byte{randomBytes(32, t), randomBytes(32, t), randomBytes(32, t)},
```

Comment:
why before it was 33 and now 32 the length of the slice?

Reply:
Before we didn't use the correct length. Now we are using correct length 33 or else HTR fails